nodejs/week2/homework/meal-sharing/src/backend/api/reviews-route.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("./../data/reviews.json");
+
+router.get("/", async (request, response) => {
+    try {
+        response.send(reviews)
+    } catch (error) {
+        throw error;
+    }
+});
+
+router.get("/:id", async (request, response) => {
+    const id = request.params.id
+    if (isNaN(parseInt(id))) {
+        return response.status(400).send('Please write an correct id number')
+    }
+    else {
+        response.send(reviews.filter(reviews => reviews.id == id)[0])
+    }
+});
+
```

Comment:
Good. 
Note, that you have 2 reviews with the same id (id: 1), make sure to correct that, so each review has one specific id.

Reply:
Ohh you are right. I didnt think that I will update my codes 👍

Reply:
I checked reviews.json file but they have different id. Or did I misunderstand you ?